src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -26,12 +26,14 @@
               v-model="orgPersonFormValid"
               v-on:submit.prevent
             >
-              <!-- Person's Name -->
+              <!-- Person name -->
               <template v-if="isPerson">
                 <label class="sub-header">Person's Name</label>
                 <p v-if="isProprietor" class="info-text mb-0">
-                  If the {{ isProprietor ? 'proprietor' : 'partner' }} has changed their legal name,
```

Comment:
Previous code would never show "partner" because it's in a v-if for proprietor only.